File: codechef/easy/atm.py
with_draw,balance = map(float,input().split())
with_draw=int(with_draw)
balance=float(balance)


if with_draw>balance or with_draw<0 or with_draw%5!=0 or balance-with_draw<.5 or balance< 0:
    print(balance)
else:
    balance-=with_draw
    balance-=0.5
    print(balance)

File: codechef/easy/ATM.c
#include<stdio.h>
int main()
{
    int withdraw;
    float balance;
    scanf(" %d %f",&withdraw,&balance);
    if(withdraw>balance-0.50 || withdraw%5 != 0)
    {
        printf("%.2f",balance);
        return 0;
    }
    else
    {
        printf("%.2f",balance-0.50-withdraw);
        return 0;
    }
}
